msteams-platform/sbs-calling-and-meeting.yml

```diff
@@ -184,7 +184,7 @@ items:
 
     1. Go to **Home**.
     1. Open your bot, which is listed in the **Recent resources** section.
-    1. Select **Channels** in the left pane and select **Microsoft Teams** <img src="~/assets/images/bots/teamsicon.png" alt="Teams icon" width="20"/> from the list of available channels.    
+    1. Select **Channels** in the left pane and select **Microsoft Teams** :::image type="content" source="~/assets/images/bots/teamsicon.png" alt-text="Screenshot showing the home page.":::.    
```

Comment:
For inline icons this is incorrect usage of Markdown syntax.

Reply:
Markdown rectified.